tensorflow/tensorflow.bzl

```diff
@@ -1296,6 +1296,45 @@ def tf_extension_linkopts():
 def tf_extension_copts():
   return []  # No extension c opts
 
+# In tf_py_wrap_cc generated libraries
+# module init functions are not exported unless
+# they contain one of the keywords in the version file
+# this prevents custom python modules.
+# This function attempts to append init_module_name to list of
+# exported functions in version script
+def _append_init_to_versionscript_impl(ctx):
+    modName=ctx.attr.module_name
```

Comment:
Please fix the naming and format of the newly added code, according to the [bzl style guide](https://docs.bazel.build/versions/master/skylark/bzl-style.html).